Data/primehack.desktop

```diff
@@ -1,10 +1,10 @@
 [Desktop Entry]
 Version=1.0
-Icon=dolphin-emu
-Exec=env QT_QPA_PLATFORM=xcb dolphin-emu
+Icon=primehack
+Exec=env QT_QPA_PLATFORM=xcb primehack
 Terminal=false
 Type=Application
 Categories=Game;Emulator;
-Name=Dolphin Emulator
+Name=Primehack
```

Comment:
Shouldn't it be "PrimeHack"? (Uppercase H)

Reply:
I can always go back and change this to PrimeHack. Didn't see the typo.